docker/bootstrap/files/readiness-probe.sh

```diff
@@ -14,8 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Add Jolokia credentials, if defined in environment.
+if [[ -z $JOLOKIA_USER ]] || [[ -z $JOLOKIA_PASSWORD ]]; then
+    USER_OPT=""
+else
+    USER_OPT="--user $JOLOKIA_USER:$JOLOKIA_PASSWORD"
+fi
```

Comment:
```suggestion
if [[ -n $JOLOKIA_USER ]] && [[ -n $JOLOKIA_PASSWORD ]]; then
    USER_OPT="--user $JOLOKIA_USER:$JOLOKIA_PASSWORD"
fi
```

Reply:
You don't really need an if/else cause it's shell scripting, if the variable doesn't exist it will be empty

Reply:
👍 I'll update.